apps/src/studio/skins.js

```diff
@@ -50,13 +65,30 @@ function loadInfinity(skin, assetUrl) {
   skin.grassy = {
     background: skin.assetUrl('background2.png')
   };
+  skin.flower = {
+    background: skin.assetUrl('background3.png')
+  };
+  skin.tile = {
+    background: skin.assetUrl('background4.png')
+  };
+  skin.icy = {
+    background: skin.assetUrl('background5.png')
+  };
+  skin.snowy = {
+    background: skin.assetUrl('background6.png')
+  };
 
   // These are used by blocks.js to customize our dropdown blocks across skins
   skin.backgroundChoices = [
     [msg.setBackgroundRandom(), RANDOM_VALUE],
     // todo - come up with better names and i18n
```

Comment:
I realize this is my todo, but we should prob either open a bug, or just address this at this point (localizing is pretty easy, not sure why I didn't just do it to start).